crates/sui-core/src/consensus_adapter.rs

```diff
@@ -226,48 +256,82 @@ impl ConsensusAdapter {
     }
 
     fn await_submit_delay(
+        &self,
```

Comment:
Because you now added `self` to this function you can remove `ourselves` argument (it can be read directly from `self`)